sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRepositoryProperties.java

```diff
@@ -15,7 +15,7 @@
 
 /** Properties of this repository. */
 @Fluent
-public class ContainerRepositoryProperties implements JsonSerializable<ContainerRepositoryProperties> {
+public final class ContainerRepositoryProperties implements JsonSerializable<ContainerRepositoryProperties> {
```

Comment:
Was this done manually or as part of code generation?

Reply:
manually :( it has flattening and autorest does not add final. is there a way to enforce it with codegen before we support making them final with stream-style serialization?